spec/plans/add_compiler_spec.rb

```diff
@@ -33,8 +47,9 @@ def allow_standard_non_returning_calls
 
       # {"install_flags"=>
       #   ["--puppet-service-ensure", "stopped",
-      #   "extension_requests:1.3.6.1.4.1.34380.1.3.13=pe_compiler", "extension_requests:1.3.6.1.4.1.34380.1.1.9813=A", "main:certname=compiler"], "server"=>"primary"}
+      #   "extenion_requests:1.3.6.1.4.1.34380.1.3.13=pe_compiler", "extension_requests:1.3.6.1.4.1.34380.1.1.9813=A", "main:certname=compiler"], "server"=>"primary"}
 
+      expect_plan('peadm::util::sync_global_hiera')
```

Comment:
What does `expect_plan` with no further arguments do?

Reply:
I believe it just ensures it is run but ignores any specifics like success, failure, or how many times it actually ran. To make is more specific, I'll add the number of times it is expected to run. The plan doesn't return anything and parameters are just source and target. Mocking it is simply required because if it isn't, the underlying tasks fail because they require reading a file off disk